migrations/migration_resource.py

```diff
@@ -0,0 +1,78 @@
+"""Migration Resource
+
+Defines a FHIR resource holding data about the latest migration by specializing
+the `fhirclient.Basic` class. A single Basic resource is maintained with the most 
+recently (successfully) run migration revision held in the single Basic.code value.
+"""
+from datetime import datetime
+
+from fhirclient.models.basic import Basic
+from isacc_messaging.models.isacc_fhirdate import IsaccFHIRDate as FHIRDate
+from isacc_messaging.models.fhir import first_in_bundle, HAPI_request
+
+class MigrationManager(Basic):
+    """Represents a FHIR resource for managing migrations."""
+    def __init__(self, jsondict=None, strict=True):
+        super(Basic, self).__init__(jsondict=jsondict, strict=strict)
+
+    def __repr__(self):
+        return f"{self.resource_type}/{self.id}"
+
+    @staticmethod
+    def create_resource(resource=None) -> 'MigrationManager':
+        """Create a new Migration Manager storing the latest applied migration id"""
+        created_time = datetime.now().astimezone().isoformat()
+
+        if resource is None:
+            resource = {
+                'resourceType': 'Basic',
+                'code': {"coding": [{"system": "http://our.migration.system", "code": None}]},
+                'created': created_time,
+            }
+
+        response = HAPI_request('POST', 'Basic', resource=resource)
```

Comment:
Although we should be able to easily search for our Basic resource given the system/code, I think it would be more consistent (and one fewer HTTP request) to PUT (upsert) a Basic resource to a consistent FHIR resource ID (eg `cirg-migration-version`)